test/unit/hashring_test.js

```diff
@@ -165,3 +165,16 @@ test('servers removed out of order result in same checksum', function t(assert)
         }
     }
 });
+
+test('hashring replica point comparator', function t(assert) {
```

Comment:
These tests seems silly now but will be extended when the comparator uses more than just the hash for comparison.